composer.json

```diff
@@ -0,0 +1,70 @@
+{
+	"name": "silverorange/site",
+	"description": "Framework for building a website.",
+	"type": "library",
+	"keywords": [ "framework" ],
+	"homepage": "https://github.com/silverorange/site",
+	"license": "LGPL-2.1",
+	"authors": [
+		{
+			"name": "Charles Waddell",
+			"email": "charles@silverorange.com"
+		},
+		{
+			"name": "Isaac Grant",
+			"email": "isaac@silverorange.com"
+		},
+		{
+			"name": "Michael Gauthier",
+			"email": "mike@silverorange.com"
+		},
+		{
+			"name": "Nathan Frederikson",
+			"email": "nathan@silverorange.com"
+		},
+		{
+			"name": "Nick Burka",
+			"email": "nick@silverorange.com"
+		},
+		{
+			"name": "Steven Garrity",
+			"email": "steven@silverorange.com"
+		}
+	],
+	"require": {
+		"php": ">=5.5.0",
+		"ext-iconv": "*",
+		"ext-openssl": "*",
+		"ext-pcre": "*",
+		"aws/aws-sdk-php": "^3.0.0",
+		"codescale/ffmpeg-php": "^2.7.0",
+		"pear/mail": "^1.2.0",
+		"pear/mail_mime": "^1.10.0",
+		"pear/net_smtp": "^1.6.3",
+		"pear/services_akismet2": ">=0.3.1",
+		"pear/text_password": "^1.1.1",
+		"silverorange/mdb2": "^2.4.1",
+		"silverorange/concentrate": ">=0.1.0",
```

Comment:
I believe there is a typo here. There is currently [no version](https://github.com/silverorange/Concentrate/releases) of this package.